WordPress/src/main/java/org/wordpress/android/ui/domains/management/details/DomainManagementDetailsWebViewClient.kt

```diff
@@ -12,6 +12,13 @@ class DomainManagementDetailsWebViewClient(
         fun onRedirectToExternalBrowser(url: String)
     }
 
+    override fun doUpdateVisitedHistory(view: WebView?, url: String?, isReload: Boolean) {
+        super.doUpdateVisitedHistory(view, url, isReload)
+        if (url != null && url != "about:blank" && !navigationDelegate.canNavigateTo(url)) {
```

Comment:
The `about:blank` addition was needed to prevent navigating back from the starting detail page.

Reply:
Thank you for the explanation.